source/Ldap.Tests/NestedGroupFinderTests.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NSubstitute;
+using Octopus.Server.Extensibility.Authentication.Ldap;
+using Octopus.Server.Extensibility.Authentication.Ldap.Model;
+using Xunit;
+
+namespace Ldap.Tests
+{
+    public class NestedGroupFinderTests
+    {
+        [Theory]
```

Comment:
I think the other extensions are Nunit, what you made you choose Xunit?

Reply:
AFAIK, we're using Xunit going forward.